ckanext/ontario_theme/templates/snippets/search_form.html

```diff
@@ -0,0 +1,41 @@
+{% ckan_extends %}
+
+{% block search_facets %}
+  {% if query or facets %}
+    <p class="filter-list">  
+    {% if query %}
+      <span>
+        <span class="facet">with the keyword(s):</span>
+        <span class="filtered pill">
+          {{query}}
+          <a href="{{ c.remove_field('q', query) }}" class="remove" title="{{ _('Remove') }}"><i class="fa fa-times"></i></a>
+        </span>
+      </span>
+    {% endif %}  
+    {% if facets %}
+        {% for field in facets.fields %}
+          <span>
+          {% set search_facets_items = facets.search.get(field)['items'] %}
+          <span class="facet">
+            {% if (not query and not loop.index0==0) or query %}
```

Comment:
When testing I noticed this doesn't work.  Seems to be a typo in `loop.index0`. Should be `loop.index`.

Needs update.

Reply:
Index0 isn't a typo (in Jinja2 documentation), and it is working for me. Could it be that we're using different libraries? To be addressed....

Reply:
After chatting about it I get it now. The and is between the facets not the items.